introduction.md

```diff
@@ -0,0 +1,2 @@
+"Hello, I'm Larkin. Just like the many that came before me, I'm here to work 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.